frontend/src/App.tsx

```diff
@@ -50,9 +74,62 @@ function App() {
               onClick: () => navigate("/settings"),
             },
             {
-              children: <Outline.ArrowRightOnRectangleIcon />,
+              children: (
+                <Tooltip
+                  keepOpenOnHover
+                  content={
+                    <Card>
+                      <H3>Account</H3>
+                      <Hr />
+                      <Grid>
```

Comment:
Not necessarily opposed to this but I'd recomment to limit the use of the grid system to the page-wide grid. But feel free to use it if it makes you life a lot easier :)